components/events/EventCardSmall.tsx

```diff
@@ -31,7 +31,7 @@ export const EventCardSmall: React.FC<EventCardSmallProps> = ({
         </h2>
 
         <div className="pt-2">
-          <EventDetails event={event} small />
+          <EventDetails event={event} small ticketProvider={""} />
```

Comment:
could make this an optional prop?

Reply:
It threw a benny with the provider colour matching object. It would work as an optional now?

Reply:
yup should do!